vnda/hooks/useCart.ts

```diff
@@ -42,6 +42,7 @@ const state = {
   update: enqueue("vnda/actions/cart/updateCart.ts"),
   addItem: enqueue("vnda/actions/cart/addItem.ts"),
   updateItem: enqueue("vnda/actions/cart/updateItem.ts"),
+  simulate: invoke.vnda.actions.cart.simulation,
```

Comment:
Should this use `enqueue` as well?

Not sure, just pattern-matching

Reply:
In this case, I try to use before, but the action does't work with the enqueue.